chromium_src/services/network/cookie_expiry_browsertest.cc

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) 2012 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <string>
+
+#include "base/bind.h"
+#include "base/test/bind.h"
+
+#include "base/path_service.h"
+#include "base/strings/stringprintf.h"
+#include "brave/components/constants/brave_paths.h"
+#include "build/build_config.h"
+#include "chrome/browser/ui/browser.h"
+#include "chrome/browser/ui/browser_commands.h"
+#include "chrome/test/base/in_process_browser_test.h"
+#include "chrome/test/base/ui_test_utils.h"
+#include "content/public/browser/storage_partition.h"
+#include "content/public/test/browser_test.h"
+#include "content/public/test/content_mock_cert_verifier.h"
+#include "net/dns/mock_host_resolver.h"
+#include "net/test/embedded_test_server/embedded_test_server.h"
+
+constexpr base::TimeDelta k4YearsInDays = base::Days(1461);
+
+class CookieExpirationTest : public InProcessBrowserTest {
```

Comment:
please don't put tests into `chromium_src` dir. `brave/browser/brave_shields` is fine for this browsertest.